README.md

```diff
@@ -163,7 +163,7 @@ The API is specified in OpenAPI 3. Use [Swagger Validator Badge](https://github.
 ### Documentation
 Documentation is sourced from the `hugo/` directory. Building the docs requires the [Hugo framework](https://gohugo.io/documentation/) with the [Clyde theme](https://github.com/DNAstack/clyde). Edit the markdown files under `hugo/content/` for content changes.
 
-Run the docs locally using `make run`, which is served at `http://localhost:1313/ga4gh-search/`. Clean up before commiting using `make clean`. 
+Run the docs locally using `make run`, which is served at `http://localhost:1313/data-connect/`. Clean up before commiting using `make clean`. 
```

Comment:
Does this need a corresponding code change in clyde?

Reply:
No - I made that change in one of the previous PRs, this is the README catching up.